requirements/docs.txt

```diff
@@ -11,7 +11,7 @@ Sphinx==3.4.3
 docutils==0.14  # pyup: ignore
 
 sphinx_rtd_theme==0.5.1
-sphinx-tabs==2.0.1
+sphinx-tabs==1.3.0
```

Comment:
This should have a `# pyup: ignore` and a comment explaining why we can't upgrade it yet linking to the issue in search extension